tests/test-dirs/polarity-search/recent.t

```diff
@@ -49,49 +51,49 @@ A few simple tests that show all the things we want to preserve or improve:
   > tr '\n' ' ' | jq '.value.entries[] | del(.info) | del(.kind)' | head -n48
```

Comment:
Let's maybe increase from 48 to whatever is required to have exactly the same elements as before?

Reply:
I removed the deprecated field from these results and also changed the `head` to a jq array slice. That should make this test less sensitive.